scss/_buttons.scss

```diff
@@ -72,6 +77,11 @@
   text-shadow: 0 -1px 0 rgba(0,0,0,0.15);
   @include gradient(#8add6d, #60b044);
   border-color: darken(#60b044, 2%);
+  
+  .counter {
+    color: #60b044;
+    background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.